index.html

```diff
@@ -0,0 +1,178 @@
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+    <link
+  rel="stylesheet"
+  href="https://cdn.jsdelivr.net/npm/swiper@8/swiper-bundle.min.css"/>
+  <script src="https://cdn.jsdelivr.net/npm/swiper@8/swiper-bundle.min.js"></script> 
+  <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.6.0/jquery.min.js"></script>
+  <script src="https://kit.fontawesome.com/88a4cee998.js" crossorigin="anonymous"></script> 
+</head>
+<body>
+  
+<div id="header">
+    <div class="container">
+        <nav>
+            Nikesh
+            <img src="image/logo.png" alt="">
+            <ul id ="sideMenu">
+                <li><a href="#header"> Home</a></li>
+                <li><a href="#about"> About</a></li>
+                <li><a href="#sliding"> Slider Section</a></li>
+                <li><a href="#contact"> Contact</a></li>
+                <li><a href="#contact"> video</a></li>
+                <i class="fas fa-times" onclick="closeMenu();"></i>
+
+            </ul>
+            <i class="fas fa-bars" onclick="openMenu();"></i>
+
+        </nav>
+
+        <div class="header-text">
+            <p class="css-3d-text">Student at ITMO University</p>
+            <p> I have a dream to be frontend Developer </p>
+        </div>
+    </div>
+</div>
+<!-- My about Section -->
+<div id="about"> 
+
+    <div class="container">
+
+        <div class="row">
+            <img id="about_img "src="media/resized.png" alt="">
+        
+            <div class="about-col-1">
+
+            </div>
+            <div class="about-col-2">
+                <h1 class=""about_me> <b>About Me</b> </h1>  
+                <p id="desc">Hi! This is me Nikesh Acharya. </p>
+                <div class="more_titles">
+                    <p class="tab-links">
+                        <p id="skills" >  </p>
+                         <p id= "Education">Education: Radiant Readers Academy (10th grade),National School of Sciences(12)
+                         ITMO Uiversity(Present) </p>
+                         <p id="Experience">Experience: SEO Analyst (Adventure discovery Trek[2020-2021])
+                        English Teacher Freelancer at Profi.ru(Present)
+                        Translator and Consultant of Crytocurrency at CWD.global(Present)
+                        Manual Money Exchanging Business (Nepal-Russia)(Present)</p>
+
+
+                    </p>
+                </div>
+            </div>
+        
+        </div>
+    </div>
+</div>
+<!-- Photo Slider Slider Section -->
+<div id="sliding">
+<section class="banner">
+    <!-- <p class ="text_slider">  </p> -->
+    <div class="imgBx">
+        <img src="media/3.jpg" class="active" alt="">
+        <img src="media/2.jpg" alt="">
+        <img src="media/4.jpg" alt="">
+        
+    </div>
+    <ul class="controls">
+        <li onclick="prevSlide()"></li>
```

Comment:
prevSlide() is not defined

Reply:
Thanks I defined it. I will deploy it as well.